src/docs/product/integrations/issue-tracking/height/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Height
+sidebar_order: 1
+description: "Learn more about Sentry's Height integration and how you can effortlessly create new Height tasks from Sentry and link Sentry issues to existing Height tasks."
+---
+
+The Height Sentry integration allows your team to record and quickly resolve critical errors by linking new or existing Height tasks to Sentry issues. Linking tasks gives your teammates visibility into updates from Sentry, and can automatically mark Sentry issues as __Resolved__ as the task status changes in Height. Additionally, set up custom alerts to create new Height tasks when key triggers you care about, like reported errors or session crashes, spike above a certain threshold.
+
+The Height integration is maintained and supported by the company that created it. For more details, see [Integration Platform](/product/integrations/integration-platform/).
```

Comment:
I'm waffling between "The Height integration is" and "This integration is" because "It's" felt strange here, but I'll defer to you @imatwawana

Reply:
I'd go with "This integration is..."